api/cluster.go

```diff
@@ -189,6 +189,36 @@ func listClusters(w http.ResponseWriter, r *http.Request, t auth.Token) (err err
 	return json.NewEncoder(w).Encode(clusters)
 }
 
+// title: provisioner cluster info
+// path: /provisioner/clusters/{name}
+// method: GET
+// consume: application/x-www-form-urlencoded
+// produce: application/json
+// responses:
+//   200: Ok
+//   401: Unauthorized
+//   404: Cluster not found
+func clusterInfo(w http.ResponseWriter, r *http.Request, t auth.Token) (err error) {
+	allowed := permission.Check(t, permission.PermClusterRead)
+	if !allowed {
+		return permission.ErrUnauthorized
+	}
+	var provCluster provTypes.Cluster
+	provCluster.Name = r.URL.Query().Get(":name")
```

Comment:
You can remove provCluster and save the name inside a string variable.

```
name := r.URL.Query().Get(":name")
```

Reply:
Thanks, Guilherme! I've already done the requested change